app/blog/2020-09-22-lit-element-and-lit-html-next-preview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "LitElement 3.0 & lit-html 2.0: Early Preview Release"
+---
+
+Today we're publishing the very firse preview releases of the next major versions of our flagship libraries, LitElement and lit-html.
+
+These releases are not yet feature complete or fully API stable, but they include most of the breaking changes that we intend to make, and most of the functionality we want to carry over from the previous versions. The releases also do not yet support legacy browsers like IE11, or browsers requiring the web components polyfills.
```

Comment:
```suggestion
These releases are not yet feature complete or fully API stable, but they include most of the breaking changes that we intend to make, and most of the functionality we want to carry over from the previous versions. These releases don't yet support legacy browsers like IE11, or browsers requiring the web components polyfills.
```

"also do not yet" is kind of awkward, and the "also" refers back to the first clause of the first sentence "not yet feature complete" not the clause that immediately precedes this sentence, which is about features being carried over.

Alternately, turn this whole paragraph around by stating the positive first:

These releases include most of the breaking changes we intend to make, and most of the functionality we want to carry over from the previous versions. They are not yet feature complete or fully API stable. Notably, they don't yet support legacy browsers like IE11, or browsers requiring the web components polyfill.

Reply:
chose your paragraph